package.json

```diff
@@ -11,7 +11,7 @@
     "hubot-redis-brain" : "0.0.4",
     "hubot-scripts"     : "^2.17.2",
     "hubot-help"        : "^0.2.2",
-    "hubot-stackstorm"  : "^0.7.0",
+    "hubot-stackstorm"  : "^0.8.0",
```

Comment:
It's not enough, there is also https://github.com/StackStorm/st2chatops/blob/master/npm-shrinkwrap.json to re-generate